apps/web/src/pages/overview/components/Staking.vue

```diff
@@ -195,12 +210,13 @@ const handleDeposit = async () => {
     // addressBalance.value = null
     formattedAmountToStake.value = ''
 
-  }, 3000)
+  }, 2500)
 
   setTimeout(() => {
     stakingActionLoader.value = false
-  }, 3500)
+  }, 3000)
 
+  currentUserStake.value = await getUserStakeAtAddress(selectedWalletAddress.value as string)
```

Comment:
Is `getUserStake` already taken? AtAddress seems redundant but I could be missing context.

Reply:
@shanejearley - `getUserStake` could mean "get user's stake across all addresses associated with them", so I just more narrowly defined it as get the stake at a particular address of theirs. With that context, lmk if you want me to still change it.

Reply:
I see. No big preference. If we call the broader cases that use summing getTotalUserStake and getTotalUserBalance that could also work.

Reply:
Ok. I will do that. 👍